CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKHitTester.java

```diff
@@ -97,8 +320,15 @@ public boolean NeedsDisplayUpdate() {
      
     @Override
     public int DisplayUpdate() {
-        // TODO Hydra: Need to reset the needsDisplayUpdate flag in here
         int ret = VK_SUCCESS;
+        
+        if (swapChainResourcesDirty) {
```

Comment:
Let's move away from grouping resources and use resource state enums for everything.

Reply:
Added a TODO for next iteration